Demo/Demo/Models/ApprovalSessionRequest.swift

```diff
@@ -0,0 +1,17 @@
+struct ApprovalSessionRequest: Codable {
+
+    let customerId: String
+    let source: Source
+    let applicationContext: ApplicationContext
+}
+
+struct Source: Codable {
+
+    let paypal: PayPalSource
```

Comment:
```suggestion
    let payPal: PayPalSource
```

Reply:
Not sure if there's a convention in place or if this is how it is in the backend JSON.

Reply:
All of this structs are a map 1:1 with the orders api.